README.md

```diff
@@ -1 +1,31 @@
-# mqperf v2
\ No newline at end of file
+# mqperf v2
+## Overview
+This repository enables you to bootstrap the AWS EKS cluster alongside the [Apache Kafka cluster by Strimzi](https://strimzi.io/) and the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack).
+
+## Prerequisites
+- Terragrunt version 0.38.7 or newer
+- Terraform version v1.2.6 or newer
+- Configure your AWS credentials using one of the supported authentication mechanisms.
+
+## Quick start - deploying all modules
```

Comment:
The plan is to have much more modules here, and deploying all of them doesn't make sense (for example - creating EKS and GKE cluster in the same time)